percy/resource.py

```diff
@@ -9,9 +9,11 @@ def __init__(self, resource_url, is_root=False, **kwargs):
         self.resource_url = resource_url
         if 'sha' in kwargs and 'content' in kwargs or not ('sha' in kwargs or 'content' in kwargs):
             raise ValueError('Exactly one of sha or content is required.')
+        # Encode the content if it exists
+        self.content = kwargs['content'].encode('utf-8') if kwargs.get('content') else None
```

Comment:
Yeah, this problem manifests only in python 3 (plus there are some other issues with JSON encoder) - see travis